syntax = "proto3";

package second.protos;

message VoxelGenerator{
    repeated float voxel_size = 1;
    repeated float point_cloud_range = 2;
    uint32 max_number_of_points_per_voxel = 3;
    bool full_empty_part_with_mean = 4;
    bool block_filtering = 5;
    int64 block_factor = 6;
    int64 block_size = 7;
    float height_threshold = 8;
}
